### DISCUSSION TITLE
360 Day Orbit

### DISCUSSION THREAD

**Discussion by Wijitmaker** [Stellarium/stellarium#1722] [Q&A]
Greetings!  I have been doing some personal research with chronology and astronomy and some of the connections between.  

What if the modern understanding of a 365.25 year is a recent phenomena?

I have been doing some reading of Immanuel Velikovsky and his book Worlds in Collision.  In it, he suggests a 360 day earth ywar was recorded by our ancestors and is evident in dozens of distinct ancient records.  He espouses a radically alternative view with catastrophist cosmology.  

My question for the team here is there a way in Stellarium to alter the reckoning of a year at a given point of time; for example at 717 BC.  I don't fully understand his ideas, but I believe he is looking at an pole shift event that is caused due to a nearby planetary body (Venus/Mars - on an erratic orbit).  So, I would assume the speed at which earth makes a complete rotation around the sun would be where the alteration takes place.  I don't know if it would have altered the the current path or the speed of rotation, but for simplicity, I would assume not.

Thanks for being open minded :)

Looking forward to your suggestions of how this might be modeled.  

PS.  Thanks for all the work you do.  I am enjoying moding the artwork of the constellations right now in an attempt to connect them to ancient zodiac symbology.  This has been a fascinating journey for me and Stellarium has been a big part of that.

**Comment by gzotti** [comment]
Sorry to disappoint you but Stellarium is developed by mainstreamists who try to follow accepted and well-formulated physical models. The open-source nature of this project allows for radical changes in forked projects to be made by somebody else though. We do not have a third life to dive into speculations.

**Reply by Wijitmaker** [reply]
No worries.  I expected that this would be the official stance.  Would you be so kind to describe (If you were to go about making these changes...) how you would go about accomplishing this task?  I would like to compile a modified local branch for my research.  Is there a specific module(s) or script that you could point me to?  Thanks for your time and consideration.

**Reply by gzotti** [reply]
Building is described in the BUILDING.md file. When building the real program succeeds, exchange the approved physical models from the astronomical literature by your own. Do not publish this under the Stellarium name, but use a different name, but give proper credits where due. Document what you have changed.
